utils/webapp_connector.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+/**
```

Comment:
nit: is the multi-line comment necessary (vs. extra `//` as necessary).

Reply:
I used ESDoc conventions on new files I created, this one included. Can convert if you'd prefer.